gapis/api/vulkan/vulkan.api

```diff
@@ -302,6 +304,7 @@ sub ref!ExtensionSet supportedDeviceExtensions() {
   supported.ExtensionNames["VK_KHR_shader_terminate_invocation"] = true
   supported.ExtensionNames["VK_EXT_blend_operation_advanced"] = true
   supported.ExtensionNames["VK_EXT_index_type_uint8"] = true
+  supported.ExtensionNames["VK_KHR_buffer_device_address"] = false
```

Comment:
I thought we weren't going to expose this extension yet, until it's completely implemented? I'm not seeing any of the todos we had talked about? It's possible I am missing something or misremembering.

Reply:
I've assumed that if it's `false`, it would mean that we wouldn't be exposing but I'll remove it from here just in case.